### ISSUE TITLE
Increase font size

### ISSUE THREAD

**Issue by LeoMcA** [mozilla/discourse#83]
Try increasing font size, try not to break _everything_.

- Medium a good example
- Try for 16px baseline text
- Try making titles bigger (2.4em?)
- 1.6em for category list
- Use invision prototypes as a 'style guide' for relative sizes of text
- *whitespace = good!*

**Comment by LeoMcA** [comment]
Improvements deployed as "moz://a beta" theme on production, to enable:

- Go to: https://discourse.mozilla.org/my/preferences/interface
- Select "moz://a beta" theme from dropdown
- Click "Save Changes"

@m-branson it'd be great to get your 👍 to show these improvements by default to all users

**Comment by comzeradd** [comment]
Nice! Threads are definitely more readable now 👓

**Comment by m-branson** [comment]
Looking great @LeoMcA! Much more legible all around.

An additional change I'd call out re: text legibility is the category badge. Recommend bringing text size up to 1em there. I've also noticed that the margin and/or padding on those is odd and appears inconsistent in spots. Recommend making text padding a consistent 2px on both left and right sides. The result is a bit of equal room on each side for the text to 'breathe' within the color highlight.

**Comment by LeoMcA** [comment]
Thanks for the feedback @m-branson! 😄 

> I've also noticed that the margin and/or padding on [category badges] is odd and appears inconsistent in spots.

That's down to subcategories using a sliver of their parent category's colour in their own box. Most of our parent categories have the same colour as their subcategory, so I just removed this altogether.

I refined things a little, and shipped to prod for all users: https://discourse.mozilla.org/t/mozilla-discourse-release-2017-08-29/18537

Further feedback welcome, whenever and wherever!
